operator/pkg/apis/dataplane/v1alpha1/dataplane.go

```diff
@@ -38,6 +38,23 @@ type DataPlaneList struct {
 }
 
 type DataPlaneSpec struct {
```

Comment:
I can look into this, but I want to integrate with constraints API once its stable and won't change, so may be once we move to v1.Contraints, I can add it, thoughts?

Reply:
You could just copy it for now if you don't want to rely directly.

Reply:
Or just rely on a version of karpenter that gets stale, and you may need to rebase eventually

Reply:
Logged an issue https://github.com/awslabs/kubernetes-iteration-toolkit/issues/57 for now, as it will require some code changes. Trying to test/fix CSR issue this week, will get back to this